dxconfgen.py

```diff
@@ -90,6 +103,8 @@ def generate_confs(blockchain, p2pport, rpcport, configname):
   parser.add_argument('-p2p', '--p2pport', type=str, help='p2pport override', required=False, default=None)
   parser.add_argument('-rpc', '--rpcport', type=str, help='rpcport override', required=False, default=None)
   parser.add_argument('-n', '--configname', type=str, help='config file name', required=False, default=None)
+  parser.add_argument('-u', '--rpcuser', type=str, help='rpc username', required=False, default=None)
+  parser.add_argument('-p', '--rpcpass', type=str, help='rpc password', required=False, default=None)
 
   args = parser.parse_args()
-  generate_confs(args.blockchain, args.p2pport, args.rpcport, args.configname)
```

Comment:
Why is the last line outside of if __name__ == main?

Reply:
Updated